server/app.js

```diff
@@ -25,6 +25,7 @@ var cookieParser = require("cookie-parser");
 var flash = require("connect-flash");
 
 
+var session = require("cookie-session");
 
```

Comment:
Forgot to remove the old one, that has been fixed. 

Previously `var session` was `var session = require("express-session")`